Source/Glitch64/OGLglitchmain.cpp

```diff
@@ -29,6 +29,21 @@
  */
 #include <Settings/Settings.h>
 
+#if !defined(_WIN32) && !defined(__ANDROID__) && !defined(ANDROID)
+/*
+ * Do not include <SDL.h> for Windows or Android builds.
+ * Most other platforms tend to include it by default, though.
+ *
+ * Just a few dependencies to SDL remain in the code:
+ *     1.  SDL_SetGammaRamp(), in CorrectGamma()
+ *     2.  SDL_SetGammaRamp(), in grGetGammaTableExt()
+ *     3.  SDL_Surface* m_pScreen = SDL_SetVideoMode(...), to create context
+ * Context creation can be done fine without SDL, and eventually it will be
+ * nice to remove it.  However, Mac & modern Linux tends to come with SDL.
+ */
+#include <SDL.h>

```

Comment:
I understand neither of us wants SDL.  I don't either; maybe you fear I'm trying to restore it.

I was not adding SDL functionality back in, so much as just trying to fix the compile errors as directly as possible.  I don't mean to call or add any SDL functions, but removing them is too immediate of an approach to me when I can just debug the smaller code it after the SDL works.

Putting this into perspective--think back to when you helped port Project64 to 64-bit x86.  You didn't _immediately_ delete/replace all of the inline assembly and other features causing 64-bit Project64 to not build; you used breakpoints or macro'd them out.  If you immediately tried replacing the 32-bit x86 inline assembly with x64 code you would have no way to debug it without getting it to compile first.

So in this case the quickest way to get it to compile is include SDL.h, and then remove SDL.

So I can take the long road and immediately replace your leftover SDL lines, but this will:
a) Not be debuggable until after everything compiles and starts to run here on Linux.
b) Take longer.  I can replace the SDL with some extra time but it will be a while before I can have this PR updated.